book/src/syntax.md

```diff
@@ -261,3 +278,29 @@ select [
     big = 5_000_000,
 ]
 ```
+
+## Keywords
+
+At the moment, PRQL uses only four keywords:
+
+- `prql`
+- `let`
+- `func`
+- `switch`
+
+If you want to use these names as your columns or relations, use backticks:
```

Comment:
I've tried to keep the "voice" consistent through the book, and done that by avoiding the 2nd person (i.e. "you"); so it sounds more descriptive than instructional.

```suggestion
To use these names as columns or relations, use backticks:
```

(not trying to refine your work, if you prefer I do a pass after, or don't change it, totally fine, lmk)

Reply:
Yup better